alf/drivers/policy_driver.py

```diff
@@ -48,19 +53,19 @@ def __init__(self,
                 callable(time_step.Trajectory).
             metrics (list[TFStepMetric]): An optional list of metrics.
             training (bool): True for training, false for evaluating
```

Comment:
no argument `training` in function signature now

Reply:
removed